odoo/addons/base/models/ir_attachment.py

```diff
@@ -110,7 +110,7 @@ def _get_path(self, bin_data, sha):
             os.makedirs(dirname)
         # prevent sha-1 collision
         if os.path.isfile(full_path) and not self._same_content(bin_data, full_path):
-            raise UserError("The attachment is colliding with an existing file.")
+            raise UserError(_("The attachment is colliding with an existing file."))
```

Comment:
Another potential English improvement

```suggestion
            raise UserError(_("The attachment collides with an existing file."))
```

Reply:
@ticodoo thanks and updated